sdk/src/main/java/io/dapr/actors/runtime/AppToDaprClientBuilder.java

```diff
@@ -18,8 +17,9 @@
      * @return Builds an async client.
      */
     public AppToDaprAsyncClient buildAsyncClient() {
```

Comment:
This class needs to go. Any place where this class is being used, should just be DaprClientBuilder.